src/commands/projects/create.ts

```diff
@@ -1,34 +1,96 @@
-import type { Config, DbConfig } from '../../interfaces/project-create-interface';
+import type {
+  Config,
+  DbConfig,
+  ProcessedArguments,
+} from '../../interfaces/project-create-interface';
 import type ForestExpress from '../../services/dumpers/forest-express';
+import type CommandGenerateConfigGetter from '../../services/projects/create/command-generate-config-getter';
 import type DatabaseAnalyzer from '../../services/schema/update/analyzer/database-analyzer';
 import type * as OclifConfig from '@oclif/config';
 
+import { flags } from '@oclif/command';
+
 import AbstractProjectCreateCommand from '../../abstract-project-create-command';
+import {
+  nosqlDbDialectOptions,
+  sqlDbDialectOptions,
+} from '../../services/prompter/database-prompts';
 
 export default class CreateCommand extends AbstractProjectCreateCommand {
   private readonly databaseAnalyzer: DatabaseAnalyzer;
 
   private readonly dumper: ForestExpress;
 
+  private readonly commandGenerateConfigGetter: CommandGenerateConfigGetter;
+
+  private readonly _agent: string | null = null;
+
+  static override readonly flags = {
+    ...AbstractProjectCreateCommand.flags,
```

Comment:
This feels strange to be honest. On the mother class, it is not specified that flags should be overridden in any place. So, no contracts are specified anywhere that flags should be on the child class. This means that you will rely on the developper comprehension, where OOP is all about specifying contracts that developper should respect. Plus, the child class is responsible for getting back the mother flags. In OOP, the mother class should have this ownership.

Reply:
It's the only way to do it. Oclif inheritance is not good.

Reply:
👌